cpp/src/io/parquet/reader_impl.cu

```diff
@@ -49,20 +49,68 @@ constexpr uint32_t PARQUET_COLUMN_BUFFER_SCHEMA_MASK          = (0xffffff);
 constexpr uint32_t PARQUET_COLUMN_BUFFER_FLAG_LIST_TERMINATED = (1 << 24);
 
 namespace {
+
+parquet::ConvertedType logical_type_to_converted_type(parquet::LogicalType const &logical)
+{
+  if (logical.__isset.STRING) {
+    return parquet::UTF8;
+  } else if (logical.__isset.MAP) {
+    return parquet::MAP;
+  } else if (logical.__isset.LIST) {
+    return parquet::LIST;
+  } else if (logical.__isset.ENUM) {
+    return parquet::ENUM;
+  } else if (logical.__isset.DECIMAL) {
+    return parquet::DECIMAL;  // TODO set decimal values
```

Comment:
is this a TODO for a future PR?

Reply:
Yes. Only decimal scale is used now. 
Other values are unused now. Like `decimalType.precision`, `IntType.bitWidth`,  `IntType.isSigned`, `TimeStampType.isAdjustedToUTC`, `TimeType.isAdjustedToUTC`.